build-parent/pom.xml

```diff
@@ -60,7 +60,7 @@
         <!-- MicroProfile TCK versions -->
         <microprofile-health-api.version>2.1</microprofile-health-api.version>
         <microprofile-config-api.version>1.3</microprofile-config-api.version>
-        <microprofile-metrics-api.version>2.2</microprofile-metrics-api.version>
+        <microprofile-metrics-api.version>2.2.1</microprofile-metrics-api.version>
```

Comment:
Do we need a new SmallRye Metrics? Or it's purely a TCK fix?

Reply:
2.2.1 has a TCK fix so that it works with the new (more correct) MP Config implementation. No action on SR-Metrics side needed.